lib/snitch-core/src/lib.rs

```diff
@@ -63,6 +63,13 @@ pub struct Store<const N: usize> {
     stored_record_count: usize,
 }
 
+#[derive(Copy, Clone, Eq, PartialEq)]
+#[cfg_attr(feature = "counters", derive(counters::Count))]
```

Comment:
I don't see an actual `counters` features in `snitch-core/Cargo.toml`, does this actually work?  (also, I thought `rustc` had started complaining if you used features that didn't exist)

Reply:
`counters` is an optional dependency:
https://github.com/oxidecomputer/hubris/blob/89519098481e332c56a28b9aab9a53609c7d01dc/lib/snitch-core/Cargo.toml#L9
an optional dependency creates an implicit feature with the same name as the dependency

Reply:
TIL!